README.md

```diff
@@ -364,6 +392,207 @@ npm run docz -- dev
 
 After compilation has finished you can access docz at http://localhost:3030/.
 
+## GraphQL API
+
+Our API is generally served via GraphQL at `/api/graphql` on your Coral installation. If you're running Coral locally, this would be https://localhost:8080/api/graphql.
+
+You can enable the GraphiQL interface at https://localhost:3000/graphiql (Note the port number here is not 8080, this is because this route is directly served by the server, and not the webpack development server) when running in development to access a GraphQL playground to use with documentation provided in the sidebar on what edges are available to you. You can do this by setting `ENABLE_GRAPHIQL=true`. **(🚨 Note 🚨) we do not recommend using this in production environments as it disables many safety features used by the application**.
+
+### Making your first request
+
+To learn a bit about how to interact with Coral, we'll query for comments on a
+page of Coral.
+
+The GraphQL endpoint we have can be used with any HTTP client available, but our
+examples below will use the common `curl` tool:
+
+```sh
+curl --request POST \
+  --url "http://localhost:8080/api/graphql" \
+  --header "content-type: application/json" \
+  --data '{"query":"query GetComments($url: String!) {story(url: $url) { id metadata { title } url title comments { nodes { id body author { id username } } } } }","variables":{"url":"http://localhost:8080/"},"operationName":"GetComments"}'
+```
+
+When you unpack that, it's really quite simple. We're executing a `POST` request
+to the `/api/graphql` route of the local Talk server with the GraphQL
+request we want to make. It's composed of the `query`, `variables`, and
+`operationName`.
```

Comment:
Is it even necessary to mention `operationName` here? It seems like an edge case. It's an optional value ONLY required when having multiple queries, which I think is an  edge case no?

Reply:
I see it's helpful for debugging.